### ISSUE TITLE
Bigger field for a comment

### ISSUE THREAD

**Issue by michal-bartak** [dbeaver/dbeaver#35447]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Please make accessing a comment easier by making its window wider at least.
Here is what it looks like by default.

![image](https://github.com/user-attachments/assets/a4b20496-dda6-4f46-9791-e03b8ab3390c)

Even expanding this part of UI vertically, the comment window is very narrow. 
It could be helpful if it occupied the whole width of the tab it belongs to.
ObjectId field could be relocated to the first row, wrapping to the new line if there is not enough space.

**Describe alternatives you've considered**
* Adding the comment DDL to the content of the source code. Somewhere between function source code and privileges.
* A modal popup with a Comment editor

**Additional context**
We document our databases using object comments. Then we generate wiki-like documentation out of it. The current size of comment windows makes it impossible to read even the first sentence of the documentation.

Please note, it applies to all objects, not only functions.

**Comment by ask9** [comment]
For me, it even looks like a bug because the same fields **sometimes** are displayed stretchable to the whole width of the tab for **tables** (dividing the whole available space of a tab by 50%/50% for left and right part of textboxes) but not for other objects: functions, columns, check constraints, indexes, references, schemas, and possibly all other objects(?).
I failed to determine the certain conditions when they are shown stretchable but looks like most of time they are not stretchable .

As a temporary workaround, change the height of that panel (but I agree its not a long term solution).


https://github.com/user-attachments/assets/04da0e84-511a-4223-9e99-fd7b49051822

**Comment by E1izabeth** [comment]
Thank you for request

**Comment by ask9** [comment]
By the way, sometimes dynamic stretching of the textbox works correctly:

https://github.com/user-attachments/assets/ecd513b5-b77a-4313-9d1f-f5275f292801

Please also consider to rearrange the textbox for description from the possible choices:

![image](https://github.com/user-attachments/assets/31e6366e-47f9-48b0-b109-7697ac157fcf)

**Comment by michal-bartak** [comment]
BTW aren't those "Procedure Name" and "Procedure Description" unnecessary long? IMO the "procedure" word could be omitted.

**Comment by ask9** [comment]
Indeed, since the type of the object has already shown in the "Type" field, it seems to be it is not neccessarily here to duplicate it again in the labels left of the textboxes stealen free space for another UI elements.

But if we take a look at other object types, we will see that some other object types don't have the "Type: <type_name>" field.
So it would be unclear for some users what type of the object is displayed (but there are still other easy ways to determine it).

Perhaps it would be benefitial to standartize this panel for all the objects (show object type for all the objects and remove it from all the labels to enlarge the area with the "comment" text?)

![изображение](https://github.com/user-attachments/assets/8893a0da-463d-47da-a923-ac364b806b9f)

![изображение](https://github.com/user-attachments/assets/d2b45e0d-88d2-413c-a773-77fffdd81932)

![изображение](https://github.com/user-attachments/assets/6a228830-2fb2-49e8-b0f3-84f729ef5818)

![изображение](https://github.com/user-attachments/assets/4baedd9c-84ee-424a-a9f4-3f8eb1d56d63)

**Comment by michal-bartak** [comment]
I know it's off-topic to continue here (unless considering it as the foundation for  optimized screen estate)
But there is a lot of inconsistency in the naming.

* At first, the object type is kind of described by its tab. There is a distinct icon for every object type.
* Then I consider, that the user knows exactly what he is looking at because he opened this object.
* I can see Procedure Description but there is a 'Comment' for a table  (BTW why Comment vs Description ;)  )

BTW Sequence name is called Table Name (I know both come from pg_class but... )

And where is the owner of the function/procedure - but it's totally out of this scope.

**Comment by michal-bartak** [comment]
I'm not sure it's been changed recently, but the text box for the function description (postgres) is even smaller!!!
Guys, please prioritize making those items bigger by default and filling the remaining space. Currently it's unusable. 

![Image](https://github.com/user-attachments/assets/78e06339-0b23-4a35-831f-68d2ed36af62)
